example/example/settings.py

```diff
@@ -46,6 +46,7 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'uaa_client.middleware.UaaRefreshMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
```

Comment:
Positioning of the middleware is very important, as it needs to be run after the session and auth middleware in particular. This will need to be documented.